css/style.css

```diff
@@ -1,6 +1,5 @@
-/* 
```

Comment:
For above file ^^ check out http://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder to ignore ds store